src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/AuthorisedGetEventTriggerOperation.java

```diff
@@ -85,7 +87,7 @@ public CaseUpdateViewEvent executeForCaseType(String caseTypeId, String eventId,
             getEventTriggerOperation.executeForCaseType(caseTypeId,
                 eventId,
                 ignoreWarning));
-
+        updateWithAccessControlMetadata(caseUpdateViewEvent, CHECK_REGION_LOCATION_TRUE);
```

Comment:
I guess we can avoid this boolean flag as it's not very deep.

Reply:
Fixed in 8e7b34d